modules/dr-restoring-cluster-state.adoc

```diff
@@ -232,6 +232,192 @@ If the status is `Pending`, or the output lists more than one running etcd pod,
 
 .. Repeat this step for each lost control plane host that is not the recovery host.
 
+. Delete and recreate other, non-recovery, control plane machines, one by one. After these machines are recreated, a new revision is forced and etcd scales up automatically.
+
+If you are running installer-provisioned infrastructure, or you used the Machine API to create your machines, follow these steps. Otherwise, you must create the new master using the same method that was used to originally create it.
```

Comment:
@tmalove This also looks like a place where the list is broken.

Reply:
Thanks @maxwelldb  I just went through each line and got the level 1 list to work for Step 12, so I will go back now and look a this. Also, I will send this for SME review and then see about compressing some of the list levels from the original writer.

Reply:
Definitely not your job to fix it, but it could definitely use some TLC if you have time.